### PR TITLE
Switched to GitHub provider due to a vender issue

### PR DESCRIPTION
**Pull request by Daz-wallace**
As recommended by the vender. See issue #360

### CODE REVIEW

File: TextMate/TextMate2.download.recipe [hunk 1]
```diff
@@ -3,75 +3,85 @@
...
+			<key>Processor</key>
+			<string>GitHubReleasesInfoProvider</string>
+			<key>Arguments</key>
+			<dict>
+				<key>github_repo</key>
+				<string>textmate/textmate</string>
+				<key>include_prereleases</key>
+				<string>%PRERELEASE%</string>
+			</dict>
+		</dict>
```
Comment (nmcspadden) [inline TextMate/TextMate2.download.recipe:34]: What's the purpose of calling this processor twice? The second one has more arguments - what exactly does this one do?

**Review by nmcspadden — CHANGES_REQUESTED** [review]
Please also provide the output of `autopkg run -vv` for this recipe.

**Comment by Daz-wallace** [comment]
```
% autopkg run -vvv /Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.download.recipe 
Processing /Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.download.recipe...
WARNING: /Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.download.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
{'AUTOPKG_VERSION': '2.1',
 'GIT_PATH': '/Applications/Xcode.app/Contents/Developer/usr/bin/git',
 'MUNKI_REPO': '/Users/darren/munki_repo',
 'NAME': 'TextMate2',
 'PARENT_RECIPES': [],
 'PRERELEASE': '',
 'RECIPE_CACHE_DIR': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2',
 'RECIPE_DIR': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.download.recipe',
 'RECIPE_REPOS': 
**[TRUNCATED],**
 'RECIPE_SEARCH_DIRS': 
**[TRUNCATED]**
 'verbose': 3}
GitHubReleasesInfoProvider
{'Input': {'github_repo': 'textmate/textmate', 'include_prereleases': ''}}
GitHubReleasesInfoProvider: No value supplied for CURL_PATH, setting default value of: /usr/bin/curl
GitHubReleasesInfoProvider: Selected asset 'TextMate_2.0.6.tbz' from release 'v2.0.6'
{'Output': {'url': 'https://github.com/textmate/textmate/releases/download/v2.0.6/TextMate_2.0.6.tbz',
            'version': '2.0.6'}}
URLDownloader
{'Input': {'filename': 'TextMate2-2.0.6.tbz',
           'url': 'https://github.com/textmate/textmate/releases/download/v2.0.6/TextMate_2.0.6.tbz'}}
URLDownloader: No value supplied for prefetch_filename, setting default value of: False
URLDownloader: No value supplied for CHECK_FILESIZE_ONLY, setting default value of: False
URLDownloader: Storing new Last-Modified header: Sat, 28 Dec 2019 06:41:05 GMT
URLDownloader: Storing new ETag header: "4fefb780c33e06372b0d7dd670f0f9d8"
URLDownloader: Downloaded /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz
{'Output': {'download_changed': True,
            'etag': '"4fefb780c33e06372b0d7dd670f0f9d8"',
            'last_modified': 'Sat, 28 Dec 2019 06:41:05 GMT',
            'pathname': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz',
            'url_downloader_summary_result': {'data': {'download_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz'},
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'downloaded:'}}}
EndOfCheckPhase
{'Input': {}}
{'Output': {}}
Unarchiver
{'Input': {'archive_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz',
           'destination_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2',
           'purge_destination': True}}
Unarchiver: Guessed archive format 'tar_bzip2' from filename TextMate2-2.0.6.tbz
Unarchiver: Unarchived /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz to /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2
{'Output': {}}
CodeSignatureVerifier
{'Input': {'input_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2/TextMate.app',
           'requirement': 'identifier "com.macromates.TextMate" and anchor '
                          'apple generic and certificate '
                          '1[field.1.2.840.113635.100.6.2.6] /* exists */ and '
                          'certificate leaf[field.1.2.840.113635.100.6.1.13] '
                          '/* exists */ and certificate leaf[subject.OU] = '
                          '"45TL96F76G"'}}
CodeSignatureVerifier: Verifying code signature...
CodeSignatureVerifier: Deep verification enabled...
CodeSignatureVerifier: Strict verification not defined. Using codesign defaults...
CodeSignatureVerifier: /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2/TextMate.app: valid on disk
CodeSignatureVerifier: /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2/TextMate.app: satisfies its Designated Requirement
CodeSignatureVerifier: /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2/TextMate.app: explicit requirement satisfied
CodeSignatureVerifier: Signature is valid
{'Output': {}}
{'AUTOPKG_VERSION': '2.1',
 'CHECK_FILESIZE_ONLY': False,
 'CURL_PATH': '/usr/bin/curl',
 'GIT_PATH': '/Applications/Xcode.app/Contents/Developer/usr/bin/git',
 'MUNKI_REPO': '/Users/darren/munki_repo',
 'NAME': 'TextMate2',
 'PARENT_RECIPES': [],
 'PRERELEASE': '',
 'RECIPE_CACHE_DIR': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2',
 'RECIPE_DIR': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.download.recipe',
 'RECIPE_REPOS': 
**[TRUNCATED]**
 'RECIPE_SEARCH_DIRS': 
**[TRUNCATED]**
 'archive_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz',
 'destination_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2',
 'download_changed': True,
 'etag': '"4fefb780c33e06372b0d7dd670f0f9d8"',
 'filename': 'TextMate2-2.0.6.tbz',
 'github_repo': 'textmate/textmate',
 'include_prereleases': '',
 'input_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/TextMate2/TextMate.app',
 'last_modified': 'Sat, 28 Dec 2019 06:41:05 GMT',
 'pathname': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz',
 'prefetch_filename': False,
 'purge_destination': True,
 'release_notes': '',
 'requirement': 'identifier "com.macromates.TextMate" and anchor apple generic '
                'and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists '
                '*/ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* '
                'exists */ and certificate leaf[subject.OU] = "45TL96F76G"',
 'url': 'https://github.com/textmate/textmate/releases/download/v2.0.6/TextMate_2.0.6.tbz',
 'url_downloader_summary_result': {'data': {'download_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz'},
                                   'summary_text': 'The following new items '
                                                   'were downloaded:'},
 'verbose': 3,
 'version': '2.0.6'}
Receipt written to /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/receipts/TextMate2.download-receipt-20201016-163815.plist

The following new items were downloaded:
    Download Path                                                                                            
    -------------                                                                                            
    /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.download.TextMate2/downloads/TextMate2-2.0.6.tbz  
darren@darren-C02WV1GDHV2R ~ % 

```

File: TextMate/TextMate2.download.recipe (hunk 1 above)
Comment (Daz-wallace) [reply TextMate/TextMate2.download.recipe:34]: That's my bad, sorry about that. Should be corrected now

**Pull request merged**